tools/benchcomp/configs/perf-regression.yaml

```diff
@@ -36,6 +36,6 @@ visualize:
         # benchmark has regressed if the lambda returns true.
         test: "lambda old, new: False if not old else not new"
       - metric: solver_runtime
-        test: "lambda old, new: False if new < 2 else new/old > 1.1"
+        test: "lambda old, new: False if new < 2 else new/old > 1.5"
```

Comment:
The new threshold 1.5 is not consistent as the description (20%). Which one is the right one?